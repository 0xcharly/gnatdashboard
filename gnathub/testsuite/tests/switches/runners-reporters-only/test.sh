gnathub --runners-only --reporters-only
