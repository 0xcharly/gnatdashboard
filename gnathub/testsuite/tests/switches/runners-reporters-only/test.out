gnathub: error: --runners-only and --reporters-only are mutually exclusive.
try "gnathub --help" for more information.
